src/modules/i18n/locales/de.json

```diff
@@ -537,6 +537,8 @@
     "search_in_catalogue_placeholder": "Suche in importierten Karten",
     "search_placeholder": "Suche nach Adressen, Parzellen oder Karten",
     "search_title": "Ort suchen oder Karte hinzufügen:",
+    "search_result_origin_kantone": "Kt.",
+    "search_result_origin_district": "Bezirk",
```

Comment:
There is already a `district` key and a `ct` key so no need for new keys just use the existing one in code